src/components/refactor/dialogs/line-creation/line-creation-dialog.js

```diff
@@ -291,7 +293,8 @@ const LineCreationDialog = ({
 
 LineCreationDialog.propTypes = {
     editData: PropTypes.object,
-    currentNodeUuid: PropTypes.string,
+    studyUuid: PropTypes.string,
+    currentNode: PropTypes.string,
```

Comment:
change type to object instead of string
```suggestion
    currentNode: PropTypes.object,
```